wandb/sdk/wandb_artifacts.py

```diff
@@ -615,7 +615,7 @@ def save(
                             tel.feature.artifact_incremental = True
                     run.log_artifact(self)
             else:
-                wandb.run.log_artifact(self)  # type: ignore
+                wandb.run.log_artifact(self)
```

Comment:
should we use `_log_artifact` here as well?

Reply:
it was just for fixing mypy errors, I'd leave it for another PR

Reply:
sounds good